application/src/utils/FolderStorage.tsx

```diff
@@ -0,0 +1,61 @@
+export default class FolderStorage {
+
+  private prefix: string;
+  public readonly parent: Storage | FolderStorage;
+
+  constructor(parent: Storage | FolderStorage, prefix?: string) {
+    this.prefix = prefix ? prefix : "";
+    this.parent = parent;
+  }
+
+  clear(): void {
+    this.keys()
+      .filter((key) => key.startsWith(this.prefix))
+      .forEach((key) => this.removeItem(key));
+  }
+  getItem(key: string): string | null {
```

Comment:
What does item mean ?
Is it files what are we storing I forgor 💀
Not sure if this is the right keyword in the context maybe I'm brain dead idk

Reply:
It's a standard in storages. this is kinda files